dev/WebView2/WebView2.cpp

```diff
@@ -124,6 +101,13 @@ void WebView2::CloseInternal(bool inShutdownPath)
         m_visibilityChangedToken.value = 0;
     }
 
+    // TODO: We do not have direct analogue for AcceleratorKeyActivated with DispatcherQueue in Islands/ win32. Please refer Task# 30013704 for  more details.
+    if (auto coreWindow = winrt::CoreWindow::GetForCurrentThread())
+    {
+        m_inputWindowHwnd = nullptr;
+        m_acceleratorKeyActivatedRevoker.revoke();
+    }
+
```

Comment:
This is a good catch. It seems like a case of a more general problem - WV2 has total of ~13 xaml handlers, most of which are not revoked on Close() - and they should be. 

The ones that are particularly concerning m_renderedRevoker are m_layoutUpdatedRevoker - we've had bugs originating from Renderd during teardown, etc.

(This could potentially be addressed in a separate change)

Reply:
I opened a task to look into this more (39649391).